apps/evm/ui/swap/ConfirmationDialog.tsx

```diff
@@ -138,7 +139,7 @@ export const ConfirmationDialog: FC<ConfirmationDialogProps> = ({ children }) =>
       : undefined,
     abi: routeProcessor2Abi,
     functionName: trade?.functionName,
-    args: trade?.writeArgs,
+    args: trade?.writeArgs as any,
```

Comment:
as any?

Reply:
I tightened the type at the source, broke the type here unfortunately, would require a lot of TS magic to fix

Reply:
Yeah dont bother this component is deprecated